src/components/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+function Navbar() {
+// This is just a dummy test for navbar. TODO Implement me later
+    return ( 
+        <nav className="navbar navbar-expand-lg bg-body-tertiary">
+            <div className="container-fluid">
+                <a className="navbar-brand" >Navbar</a>

```

Comment:
**This comment can be safely ignored.** 

But if you want to flex with this project you can add an internationalization library like [I18next](https://react.i18next.com/). Even if you have support only for English, it will show that you're thinking ahead and most frontend projects have **i18n** (internationalization)
![flex](https://github.com/DimitarIt99/BookHeaven/assets/51404831/bc5a4b45-de9b-45a2-be20-05f2544623c3)

Reply:
Great, how should I work with locales\en with the entire site ? 
Should each of my component have a directory in locales\en ( 
locales\en\Book\translation.json and then locales\en\Author\translation.json  
or should the json itself contain information for what component it is  { book.titleText: "Some text", author.listText: "some list text"}
)

@androIvanov